lib/Arch/Sleigh/PPC.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022-present Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <remill/Arch/Name.h>
+#include <remill/BC/ABI.h>
+#include <remill/BC/Util.h>
+#include <remill/BC/Version.h>
+#include <remill/OS/OS.h>
+
+#include "Arch.h"
+
+namespace remill::sleighppc {
+
+class SleighPPCDecoder final : public remill::sleigh::SleighDecoder {
+ public:
+  SleighPPCDecoder(const remill::Arch &);
+
+  llvm::Value *LiftPcFromCurrPc(llvm::IRBuilder<> &, llvm::Value *,
+                                size_t) const override;
+
+  void InitializeSleighContext(
+      remill::sleigh::SingleInstructionSleighContext &,
+      const std::map<std::string, uint64_t> &) const override;
```

Comment:
Add arguments, since there are no type aliases it can be kinda hard to know what argument is what.

Reply:
[111f0b3](https://github.com/lifting-bits/remill/pull/645/commits/111f0b309f4a152c00cebdcc879130858fb1887e)

Using a type alias now so it should be clear.